server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -160,6 +160,7 @@ private static Version parseUnchecked(String version) {
     public static final IndexVersion SYNTHETIC_SOURCE_STORE_ARRAYS_NATIVELY_SCALED_FLOAT = def(9_020_0_00, Version.LUCENE_10_1_0);
     public static final IndexVersion USE_LUCENE101_POSTINGS_FORMAT = def(9_021_0_00, Version.LUCENE_10_1_0);
     public static final IndexVersion UPGRADE_TO_LUCENE_10_2_0 = def(9_022_00_0, Version.LUCENE_10_2_0);
+    public static final IndexVersion UPGRADE_TO_LUCENE_10_2_1 = def(9_023_00_0, parseUnchecked("10.2.1"));
```

Comment:
Shouldn't this constant be available in the snapshot version of Lucene?

Reply:
Let me check